registry/extension/oras/artifactmanifesthandler.go

```diff
@@ -5,11 +5,19 @@ import (
 	"encoding/json"
 	"errors"
 	"path"
+	"time"
 
 	"github.com/distribution/distribution/v3"
 	dcontext "github.com/distribution/distribution/v3/context"
 	"github.com/distribution/distribution/v3/registry/storage/driver"
 	"github.com/opencontainers/go-digest"
+	v1 "github.com/oras-project/artifacts-spec/specs-go/v1"
+)
+
+var (
+	errInvalidArtifactType      = errors.New("artifactType invalid")
+	errInvalidMediaType         = errors.New("mediaType invalid")
+	errInvalidCreatedAnnotation = errors.New("failed to parse created time")
```

Comment:
Should these errors be of type `distribution.ErrManifestVerification`?

Reply:
Actually, I don't think so. I tried to copy the structure of the existing schema2manifesthandler. the `distribution.ErrManifestVerification` error type is just an array of errors that you add to. For example, the schema2 [manifesthandler](https://github.com/oras-project/distribution/blob/be08d24a58362810abb43ba4830e2942f8d55cfe/registry/storage/schema2manifesthandler.go#L72) also adds to this error array and defines errors [here](https://github.com/oras-project/distribution/blob/rc1/registry/storage/schema2manifesthandler.go#L17). Should I change the error types anyways?